### DISCUSSION TITLE
Consulta

### DISCUSSION THREAD

**Discussion by ricarto1** [Launch-X-Latam/MisionFrontEnd#509] [Team Posts]
Hola pasaba por aquí porque no tengo idea de cómo acceder a las clases no tengo equipo y estoy de verdad perdido, alguien que me pueda orientar y gracias de antemano

**Comment by Kasspex** [comment]
X2

**Comment by Ricardo-Vargas-Gonzalez** [comment]
Hola, para acceder a las clases revisa el correo personal con el que accediste a launch X hai te dan el acceso para la misión Backend y las clases y los equipos son para el hackathon que se hará al final de Launch X por eso tanto mensaje por aqui, no se preocupen aún hay tiempo para hacer equipos, algunos todavía no tenemos equipo.

**Comment by frankYanez** [comment]
Muchas graciaa por tu info. No me deja acceder con el mail y la contraseña que me dieron. Sabes quien es el “sherpa” del que hablan? O si tienes su correo por favor? Gracias

**Comment by ricarto1** [comment]
> 

Consulta estúpida mia número 2, donde me meto para verificar los mensajes del mail que mandó launch x?

**Comment by mafer23** [comment]
El correo que te enviaron de teams

**Comment by ricarto1** [comment]
Vamos por parte, yo tengo el correo que termina en @innovacion.mx, ese correo dónde lo meto? En teams?
